force_model.py

```diff
@@ -0,0 +1,332 @@
+#converting matlab atmospheric drag to python
+
+#imports here
+#libraries
+import math
+import numpy as np
+import scipy.io
+from numpy.linalg import matrix_power
+
+##first section, Example inputs
+#gps time, to start?
+t_gps = 0
+recef = [6778137, 0, 0] #(m)

```

Comment:
```suggestion
recef = (6378137 + 400e3) * np.array([1, 0, 0]) #(m)

```
I'd keep it consistent with the Matlab code. This is due to the altitude of the spacecraft which could be varied later on.